requirements.txt

```diff
@@ -1,4 +1 @@
-setuptools
-numpy>=1.10
-llvmlite>=0.36.0dev0,<0.37
-argparse
+-e .
```

Comment:
Thanks for the patch. The code change here looks like it replicates what is suggested in the documentation etc. Just need to test this PR manually to make sure it works as expected.

Reply:
farm job `numba_wheel_29` tested this PR and expectedly failed with the appropriate error:

```
++ pip install --extra-index-url https://pypi.anaconda.org/sklam/simple -r requirements.txt
Looking in indexes: https://pypi.org/simple, https://pypi.anaconda.org/sklam/simple
Obtaining file:///src (from -r requirements.txt (line 1))
ERROR: Could not find a version that satisfies the requirement llvmlite<0.37,>=0.36.0.dev0 (from numba==0.53.0.dev0+205.g35ce74da2.dirty->-r requirements.txt (line 1)) (from versions: 0.2.0, 0.2.1, 0.2.2, 0.4.0, 0.5.0, 0.6.0, 0.7.0, 0.8.0, 0.9.0, 0.10.0, 0.11.0, 0.12.0.1, 0.12.1, 0.13.0, 0.14.0, 0.15.0, 0.16.0, 0.17.0, 0.17.1, 0.18.0, 0.19.0, 0.20.0, 0.21.0, 0.22.0, 0.23.0, 0.23.2, 0.24.0, 0.25.0, 0.26.0, 0.27.0, 0.27.1, 0.28.0, 0.29.0, 0.30.0, 0.31.0, 0.32.0, 0.32.1, 0.33.0rc1, 0.33.0, 0.34.0rc1, 0.34.0, 0.35.0rc2, 0.35.0rc3)
ERROR: No matching distribution found for llvmlite<0.37,>=0.36.0.dev0 (from numba==0.53.0.dev0+205.g35ce74da2.dirty->-r requirements.txt (line 1))
```
because no wheels were uploaded for `llvmlite<0.37,>=0.36.0.dev0` yet.

With the error message, we know that the `requirement.txt` has successfully redirected to `setup.py` where the `llvmlite` requirement is set.

Reply:
Great, thanks for confirming.